src/components/control/multi-autocompleter.jsx

```diff
@@ -13,10 +13,17 @@ const MultiAutocompleter = forwardRef(function MultiAutocompleter(props, ref) {
   const { execute: fetchSelectedChoices } = useFetch();
   const [loading, setLoading] = useState(true);
   const [options, setOptions] = useState([]);
-  const [value, setValue] = useState(props.value);
+  const mounted = useRef(false);
+
+  function arrayWrap(v) {
+    return Object.prototype.toString.call(v) !== '[object Array]' ? [v] : v;
```

Comment:
I think this is more standard but I haven't looked it up in a hot minute. 

```suggestion
    return Array.isArray(v) ? v : [v];
```